### ISSUE TITLE
pgAdmin starts but doesnt open

### ISSUE THREAD

**Issue by Bokizz** [pgadmin-org/pgadmin4#7790]
A clear and concise description of what the bug is:
I start pgAdmin 4 but it just doesnt open or show up it stays there and i cant quit it also without force quitting it 

1. Start pgAdmin 4 

Expected behavior:
The app to open and show up

 - OS: [macOS monterey ]
 - Version: [12.7.6]
 - Mode: [Desktop]

**Comment by yogeshmahajan-1903** [comment]
@Bokizz 
Can you please provide version for pgadmin? Also screenshot if any error shown? Does it shows only splash screen or any window with error?

**Comment by Bokizz** [comment]
@yogeshmahajan-1903 im using pgAdmin 4 version 8.10, no errors appear pgAdmin starts the pop up closes and it just stays in my Dock and doesnt open and i can only shut it down through force quit

**Comment by yogeshmahajan-1903** [comment]
@Bokizz 
Are you able to see pgAdmin 4 main menu? If yes, kindly share logs by clicking view logs button as shown below - 

<img width="847" alt="Screenshot 2024-08-08 at 4 28 01 PM" src="https://github.com/user-attachments/assets/628869fe-feaf-4d15-baa4-84feeb15fc17">

**Comment by Bokizz** [comment]
This is what i got from the logs
pgAdmin Runtime Environment
--------------------------------------------------------
Python Path: "/Applications/pgAdmin 4.app/Contents/Frameworks/Python.framework/Versions/Current/bin/python3"
Runtime Config File: "/Users/pc3d/Library/Application Support/pgAdmin 4/config.json"
Webapp Path: "/Applications/pgAdmin 4.app/Contents/Resources/web/pgAdmin4.py"
pgAdmin Command: "/Applications/pgAdmin 4.app/Contents/Frameworks/Python.framework/Versions/Current/bin/python3 -s /Applications/pgAdmin 4.app/Contents/Resources/web/pgAdmin4.py"
Environment: 
  - USER: pc3d
  - MallocNanoZone: 0
  - __CFBundleIdentifier: org.pgadmin.pgadmin4
  - COMMAND_MODE: unix2003
  - LOGNAME: pc3d
  - PATH: /usr/bin:/bin:/usr/sbin:/sbin:/usr/local/bin
  - SSH_AUTH_SOCK: /private/tmp/com.apple.launchd.svhKXUo22p/Listeners
  - SYSTEM_VERSION_COMPAT: 0
  - SHELL: /bin/zsh
  - HOME: /Users/pc3d
  - __CF_USER_TEXT_ENCODING: 0x1F5:0x0:0x2
  - TMPDIR: /var/folders/xq/gtlm286j26qb9g61cf7tctrm0000gn/T/
  - XPC_SERVICE_NAME: application.org.pgadmin.pgadmin4.12886270252.12886285416
  - XPC_FLAGS: 0x0
  - ORIGINAL_XDG_CURRENT_DESKTOP: undefined
  - ELECTRON_ENABLE_SECURITY_WARNINGS: false
  - PGADMIN_INT_PORT: 50448
  - PGADMIN_INT_KEY: ccb6aab2-48cc-43a6-88e1-bdb751871e02
  - PGADMIN_SERVER_MODE: OFF
--------------------------------------------------------

Total spawn time to start the pgAdmin4 server: 0.023 Sec
2024-08-08 13:25:14,044: WARNING	werkzeug:	Werkzeug appears to be used in a production deployment. Consider switching to a production web server instead.

 * Serving Flask app 'pgadmin'
 * Debug mode: off

------------------------------------------
Total time taken to ping pgAdmin4 server: 9.061 Sec
------------------------------------------
Total launch time of pgAdmin4: 9.702 Sec
------------------------------------------
Application Server URL: http://127.0.0.1:50448/?key=ccb6aab2-48cc-43a6-88e1-bdb751871e02

**Comment by yogeshmahajan-1903** [comment]
@Bokizz 
Can you please provide information from About dialogue. Main Menu Help > About pgAdmin4.
Also have you downloaded correct installer as per your machine architecture?

**Comment by Shelicus** [comment]
We have the same Problem. Do you have a Proxy? Because we can install it on a Computer outside our Network, but we have problems with Computers inside the Network.

**Comment by adityatoshniwal** [comment]
Hi @Bokizz @Shelicus try removing directory `/Users//Library/Application Support/pgAdmin 4` and then start pgAdmin.

**Comment by Shelicus** [comment]
> Hi @Bokizz @Shelicus try removing directory `/Users/<user>/Library/Application Support/pgAdmin 4` and then start pgAdmin.

It's still not working :/
I delted the folder `/Users/<user>/AppData/Roaming/pgadmin4

**Comment by adityatoshniwal** [comment]
> > Hi @Bokizz @Shelicus try removing directory `/Users/<user>/Library/Application Support/pgAdmin 4` and then start pgAdmin.
> 
> It's still not working :/ I delted the folder `/Users/<user>/ AppData/Roaming/pgadmin4`

Did you replace the <user> with your system username?

**Comment by Shelicus** [comment]
> > > Hi @Bokizz @Shelicus try removing directory `/Users/<user>/Library/Application Support/pgAdmin 4` and then start pgAdmin.
> > 
> > It's still not working :/ I delted the folder `/Users/<user>/ AppData/Roaming/pgadmin4`
> 
> Did you replace the <user> with your system username?

Of course I did. I think there is something blocked from our Network. I am not sure if it is a a Bug of the pgAdmin or postgres

**Comment by rodgermoore** [comment]
Please check if its a UI problem. Maybe its this: https://github.com/pgadmin-org/pgadmin4/issues/7781

**Comment by felipeduarte26** [comment]
same problem, I use macOS monterey 14.6.1

**Comment by khushboovashi** [comment]
Check https://github.com/pgadmin-org/pgadmin4/issues/7781

**Comment by akshay-joshi** [comment]
Hi Bokizz 

Can you please check on the latest version? Electron seems to have fixed the issue.

**Comment by akshay-joshi** [comment]
There is no response from the author, hence closing it.

**Comment by Shelicus** [comment]
Yeah, now it is working without Problems 👍

**Comment by mwpowellhtx** [comment]
I was investigating the same issue, was not starting under `W10 x64 Pro`, (not) running `pgAdmin 4 v8`, as far as I know; did not matter what I tried, I restarted both my `pg` Windows Services `16` and `14`, `16` for which I have database catalog and tables in particular, I am working with at the moment. Both of which restarted, neither of which helped. About to restart my computer, but would be helpful if I understood better if/where any `pgAdmin` logs were being documented.
